### ISSUE TITLE
[@SerenityWithCucumber] There is inconsistency in cucumber features representation if feature files defined under the same runner class.

### ISSUE THREAD

**Issue by nstbushneva** [Invictum/serenity-reportportal-integration#61]
In some cases, our test automation framework requires the sequential execution of cucumber features. In order to achieve this, we explicitly define these features under the same runner class: 
```java
@RunWith(CucumberWithSerenity.class)
@CucumberOptions(
    plugin = {"pretty"},
    features = {"src/test/resources/basic_cucumber_test.feature",
                "src/test/resources/basic_cucumber_test2.feature"})
```
When the Report Portal represents all features on the Suite view, it shows only one of the specified files, e.g. "The 2nd dummy scenario" [src/test/resources/basic_cucumber_test2.feature], which has all tests from both files [basic_cucumber_test and basic_cucumber_test2] behind. 
![screen shot 2019-03-06 at 5 46 45 pm](https://user-images.githubusercontent.com/36731862/53894709-3d767e00-4039-11e9-8b94-674a4575e619.png)

Our project has configured as *Serenity-Cucumber-jvm*: 
* serenity-reportportal-integration [1.3.0]
* maven-failsafe-plugin [2.22.1]
* serenity-core [2.0.20]
* serenity-cucumber [1.9.21]

Here is a sample project to reproduce the issue: [simple-test-framework-with-serenity-and-cucumber.zip](https://github.com/Invictum/serenity-reportportal-integration/files/2937113/simple-test-framework-with-serenity-and-cucumber.zip)

Would it be possible for you to fix this data representation issue? 
Please let me know if you need further information. Thank you! 

Basically, the same issue happens when we execute several feature files under different runner classes but in parallel.

**Comment by Invictum** [comment]
@nstbushneva 
Thanks for issue report. I looked into it and yes it is possible to fix it.

**Comment by Invictum** [comment]
Will be fixed in scope of https://github.com/Invictum/serenity-reportportal-integration/issues/62

**Comment by Invictum** [comment]
Fixed. Will be available in version 1.4.0+
